README.rst

```diff
@@ -2,121 +2,75 @@
  OpenSlides
 ============
 
-I. What is OpenSlides?
-======================
+What is OpenSlides?
+===================
 
 OpenSlides is a free web-based presentation and assembly system for
 displaying and controlling of agenda, motions and elections of an assembly.
```

Comment:
use our new sentence from new website:
"OpenSlides is a free, web based presentation and assembly system for managing and projecting agenda, motions and elections of an assembly."